src/stores/Binding/types.ts

```diff
@@ -106,6 +106,10 @@ export interface BindingState
     backfillSupported: boolean;
     setBackfillSupported: (val: BindingState['backfillSupported']) => void;
 
+    // Control sourceCapture optional settings
+    sourceCaptureTargetSchemaSupported: boolean;
+    sourceCaptureDeltaUpdatesSupported: boolean;
```

Comment:
Why were these pieces of state added to the bindings store? If you explained this before the holiday, would you mind jogging my memory?

Reply:
They are settings based on the resource config. Eventually we'll probably be adding support to mass change these settings within the bindings as well.